actor/src/main/scala/org/apache/pekko/dispatch/AbstractDispatcher.scala

```diff
@@ -329,7 +329,7 @@ abstract class MessageDispatcher(val configurator: MessageDispatcherConfigurator
   /**
    * INTERNAL API
    */
-  @inline protected[pekko] final val isThroughputDeadlineTimeDefined = throughputDeadlineTime.toMillis > 0
+  protected[pekko] final val isThroughputDeadlineTimeDefined = throughputDeadlineTime.toMillis > 0
```

Comment:
This inline is removed because its pointless, since `isThroughputDeadlineTimeDefined` is a val it will immediately get evaluated when class is initialized to a constant.